src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -155,7 +155,17 @@ private static string GetCorrectCasedPath(string path)
                     }
                     else
                     {
-                        exactPath = Directory.GetFileSystemEntries(exactPath, item).First();
+                        // exactPath = Directory.GetFileSystemEntries(exactPath, item).First();
```

Comment:
Since this change is, as far as I understand, a workaround for 'some file system issue prevents correcting the casing' - it would be good to also have comments, why this is written like this - basically add (short version of) PR description in here.

Reply:
Will remove this old commented out line of code.

Will add a comment.